src/python/WMCore/Services/DBS/DBS3Reader.py

```diff
@@ -42,7 +42,7 @@ def remapDBS3Keys(data, stringify=False, **others):
                'block_name': 'BlockName', 'lumi_section_num': 'LumiSectionNumber'}
 
     mapping.update(others)
-    formatFunc = lambda x: encodeUnicodeToBytes(x) if stringify else x
+    formatFunc = lambda x: decodeBytesToUnicode(x) if stringify else x
```

Comment:
From a quick scan, this is what worries me the most. 

In any case, the solution in #10580 should work here. Before #10289 we converted unicode to bytes and we were using py2. I assumed this was the case, for py3 also, but i ignored our beloved "native string" approach. so we can use `encodeUnicodeToBytes(x, condition=PY2 and stringify)` here, so that we keep the same behavior in py2.

Forcing unicode in py2 also is not causing any unittest to fail, but feels risky to me. So, i propose we merge #10580, then we merge this PR and we should be good to go

Reply:
since i am not sure yet about how to properly deal with a problem with workqueue in #10580, If you want to quickly merge this I would just replace this line with 

```diff
+from Utils.PythonVersion import PY2
...
- formatFunc = lambda x: decodeBytesToUnicode(x) if stringify else x
+ formatFunc = lambda x: encodeUnicodeToBytesConditional(x, condition=PY2 and stringify)
```

Reply:
Okay, I have recreated the list of DBS mock data and just pushed it in (without updating this lambda function). Let's first see how it goes. Thanks for your comments, Dario.